modules/tensor_mechanics/src/materials/LAROMANCEStressUpdateBase.C

```diff
@@ -668,18 +668,18 @@ LAROMANCEStressUpdateBaseTempl<is_ad>::computeTileWeight(
           else
           {
             // Set input to edge of limit so it is found later in computePartitionWeights
-            input = _input_limits[p][t][in_index][0];
+            input = _input_limits[p][t][in_index][1];
```

Comment:
store value of input before setting it to limit value (for later console print message):
```
GenericReal<is_ad> input_before_set_to_limit = input;
input = _input_limits[p][t][in_index][1];
```

Reply:
No, I don't think I need that, I can just do `input = _input_limits[p][t][in_index][1]` after I make the msg, no?

Reply:
jep, that works. Also in the lower limit case..